### PR TITLE
Dom ontology browser

### PR DESCRIPTION
**Pull request by terazus**

### CODE REVIEW

**Comment by hoseinmirian** [comment]
image

When exporting as jpg, the full data labels are not included. please correct me if it is normal. thanks

**Comment by hoseinmirian** [comment]
Also please check the branch as a suggested branch to improve the UI of tree view:
https://github.com/FAIRsharing/fairsharing.github.io/pull/1326

File: src/components/Ontologies/TermDetails.vue [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+          :no-data-text="`Cannot find any record with this ${selectedOntology}`"
+        >
+          <template #[`item.name`]="{ item }">
+            <div
+              :class="color + '--text'"
+              class="noBreak"
+            >
+              <a
+                class="d-flex align-center cursor-pointer"
+                :href="`/#/${item.id}`"
```
Comment (knirirr) [inline src/components/Ontologies/TermDetails.vue]:
```suggestion
                :href="`/${item.id}`"

```

**Review by knirirr — APPROVED** [review]
Looks good to me, although that `#` will need to be removed from the links to records.

**Pull request merged**